index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Week1</title>
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href=""> <img src="images/swiggylogo.png" alt = "swiggylogo" class = "swiggy-logo"> </a></li>
+                <li> <a href="" class="setloc">Set location</a> </li>
```

Comment:
Don't couple your class names with the html text. Give contextual class names which represent the overall context of the container.

Also, in case of multiple words, hyphenate the class names